euth/reports/emails.py

```diff
@@ -0,0 +1,72 @@
+from django.contrib.auth import get_user_model
+from django.contrib.sites.models import Site
+from django.core import urlresolvers
+from django.db.models import ForeignKey
+from django.template import Context
+
+from euth.contrib import emails
+
+
+def send_email_to_moderators(request, report):
+
+    obj = report.content_object
+
+    if obj:
+        name = obj._meta.verbose_name.title()
+
+        try:
+            admin_url = urlresolvers.reverse("admin:%s_%s_change" %
+                                             (obj._meta.app_label,
+                                              obj._meta.model_name),
+                                             args=(obj.pk,))
+            admin_url = request.build_absolute_uri(
+                admin_url)
+        except:
```

Comment:
The problem here is, that as this is a model with generic FK, it could theoretically happen that there is no representation in the admin interface for this model and the reverse call would fail. But you are right I will change this.